fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/logical/LogicalWindow.java

```diff
@@ -163,9 +163,8 @@ public int hashCode() {
      * Check the window function. There are some restrictions for window function:
      * 1. The number of window function should be 1.
      * 2. The window function should be one of the 'row_number()', 'rank()', 'dense_rank()'.
-     * 3. The window type should be 'ROW'.
```

Comment:
The `rank` function will default use the `range`. And if the window frame is 'UNBOUNDED' to 'CURRENT', it's the same to use `range` or `row`.